test/unit/globecontrol.js

```diff
@@ -180,10 +180,10 @@ describe('GlobeControls', function () {
 
     it('mouse wheel', function () {
         const startRange = controls.getRange();
-        event.wheelDelta = -10;
+        event.deltaY = -10;
         controls.onMouseWheel(event);
         assert.ok(controls.getRange() > startRange);
```

Comment:
Could you please switch the `>` for a `<` ? 
The sign of `deltaY` for a `wheel` event is the opposite of the `wheelDelta` for a `mouseWheel` event. Having 
```js
event.deltaY = -10
```
on line 183 means we are now zooming in, thus the range after zooming shall be smaller than before.